src/modules/FtpServer.rb

```diff
@@ -717,13 +722,31 @@ def Write
       Builtins.sleep(@sl)
 
       return false if PollAbort()
-      write_daemon
+
+      result = save_status
+
       # Progress finished
       Progress.NextStage
       Builtins.sleep(@sl)
 
       return false if PollAbort()
-      true
+
+      result
+    end
+
+    # Saves service status (start mode and starts/stops the service)
+    #
+    # @note For AutoYaST and for command line actions, it uses the old way
+    #   for backward compatibility, see {#write_daemon}. When the service
+    #   is configured by using the UI, it directly saves the service, see
+    #   {Yast2::SystemService#save}.
+    def save_status
```

Comment:
In the `http-sever` I used [`save_service`](https://github.com/yast/yast-http-server/pull/32/files#diff-85374a49b31f55190044366a9834e490R511) for identical method. What do you about use the same naming, when possible, for the "service widget adaptation"?

I don't mind to change `save_service` to  `save_status` if you think it is more accurate :)

Reply:
Well, in this case I named it `#save_status` because it saves the service status in both different ways depending on the mode. It not always "saves the service", when "service" means our SystemService.